src/test/testdata/lsl/shiftSquareBrackets.lsl

```diff
@@ -0,0 +1,29 @@
+libsl "1.0.0";
+library simple;
+
+types {
+    int(int32);
+}
+
+automaton a: int {
+
+    state S1;
+    state S2;
+    state S3;
+
+    shift S1 -> S2 by foo;
+    shift S1 -> S2 by [fix, foo,];
+    shift S1 -> S2 by [foo];
+    shift S1 -> S2 by [foo,];
+    shift S1 -> S3 by [sum(int, int), min(int), ];
+    shift S1 -> S3 by sum(int, int);
+
+    shift (S1, S2) -> S3 by foo;
+    shift (S1, S2) -> S3 by [sum(int, int), min(int)];
+
+
+    fun foo();
+    fun sum(i: int, j: int);
+    fun min(i: int);
+    fun fix();
+}
```

Comment:
Please, add an empty line at the end of file. And enable automatically trailing line breaks in Settings (ctrl+alt+S)

![image](https://user-images.githubusercontent.com/21011268/227708713-127748ca-91e9-4849-a3d8-7f45d246850e.png)

Reply:
Done